javascript/index.js

```diff
@@ -1,10 +1,32 @@
-var weather = 'lol';
-document.getElementById('date').innerHTML = new Date().toDateString();
+let lol;
 
-const response = fetch('https://api.openweathermap.org/data/2.5/weather?q=London&appid=f43223591fa77f00dda54c1439ff73ff');
+function display(lol, id, m) {
+  document.getElementById(`${id}`).innerHTML = `${m}:${lol}`;
+}
 
-response.then((json => {
-          console.log(json.json());
-				window.weather = json;
-     })
-)
+function weather(city) {
+  const settings = {
+    async: true,
+    crossDomain: true,
+    url: `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=f43223591fa77f00dda54c1439ff73ff`,
+    method: 'GET',
+  };
+
+  $.ajax(settings).done((response) => {
+    lol = response;
+    display(lol.main.temp, 'date', 'C*');
+    display(lol.name, 'country', 'Country');
+    display(lol.weather[0].main, 'weather', 'Climate');
+  });
+}
+
+function formTemp() {
```

Comment:
Use arrow functions instead of the `function` keyword

Reply:
in this comment "You should use either promises or async/await (screenshot from Odin)" you can read what is on the image and realize that it says feel free to use and not use